components/Form.js

```diff
@@ -35,38 +36,61 @@ export default function Form({ handleAddNewWord }) {
   }
   return (
     <StyledForm onSubmit={handleSubmit} ref={formRef}>
-      <StyledInput
-        name="word"
-        placeholder="...some word"
-        type="text"
-        aria-label="Enter a new word"
-        onBlur={getTranslation}
-      />
-      <StyledInput
-        name="translation"
-        placeholder="...translation"
-        type="text"
-        aria-label="Enter the translation"
-      />
-      <StyledButton aria-label="Add word to your vocabulary" type="submit">
-        ＋
-      </StyledButton>
+      <InputContainer>
+        <StyledInput
+          name="word"
+          placeholder="...some word"
+          type="text"
+          aria-label="Enter a new word"
+          //onBlur={getTranslation}
+        />
+        <StyledButton
+          aria-label="Get translation"
+          type="button"
+          onClick={getTranslation}
+        >
+          <Icon variant="arrowRight" size={20} />
+        </StyledButton>
+      </InputContainer>
+      <InputContainer>
+        <StyledInput
+          name="translation"
+          placeholder="...translation"
+          type="text"
+          aria-label="Enter the translation"
+        />
+        <StyledButton aria-label="Add word to your vocabulary" type="submit">
+          <Icon variant="checkmarkBig" size={20} />
+        </StyledButton>
+      </InputContainer>
     </StyledForm>
   );
 }
+const InputContainer = styled.div`
+  align-self: stretch;
+  display: flex;
+  gap: 0.3em;
+`;
 const StyledForm = styled.form`
   display: flex;
+  flex-direction: column;
+  align-items: center;
   justify-content: center;
+  max-width: 430px;
+  gap: 0.3em;
+  margin: 0 auto;
+  @media (min-width: 431px) {
+    flex-direction: row;
+  }
 `;
 const StyledButton = styled.button`
```

Comment:
What about our own button component from 'Button.js'?

Reply:
I didn't get it with our Button component. It has such specific styles, I did not get the styling I want.